### PR TITLE
[Feature][Feature/AES-1789] IconLink

### PR DESCRIPTION
**Pull request by zack76**
**New components: Icon Link for further use in Footer component**

Screen Recording 2020-09-18 at 1 49 08 pm

### CODE REVIEW

File: src/components/IconLink/IconLink.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+};
+
+IconLink.defaultProps = {
+  altText: undefined,
+  className: undefined,
+  height: 21,
+  icon: undefined,
+  iconImageClassName: undefined,
+  id: undefined,
+  openInANewWindow: true,
```
Comment (beau-charman) [inline src/components/IconLink/IconLink.js:56]: can we change the prop to `hasTargetInNewWindow` for boolean naming and Hyperlink prop name consistency?

File: src/components/IconLink/IconLink.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      title={altText}
+      url={target}
+    >
+      <Image
+        altText={altText}
+        className={iconImageClassName}
+        large={icon.small}
+        medium={icon.small}
+        small={icon.small}
+        style={{ height: `${height}px`, width: `${width}px` }}
```
Comment (beau-charman) [inline src/components/IconLink/IconLink.js:31]: Do these values need to change with media queries?

**Review by beau-charman — CHANGES_REQUESTED** [review]

File: src/components/IconLink/IconLink.js (hunk 1 above)
Comment (zack76) [reply src/components/IconLink/IconLink.js]: updated

File: src/components/IconLink/IconLink.js (hunk 2 above)
Comment (zack76) [reply src/components/IconLink/IconLink.js:31]: updated

**Comment by kevin-ho87** [comment]
Just a curious question. what happens when different images with different heights are added? how would they line up?

**Review by kevin-ho87 — APPROVED** [review]

**Pull request merged**